Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3208,6 +3209,9 @@ void WebPageProxy::handleMouseEvent(const NativeWebMouseEvent& event)
     UNUSED_PARAM(didRemoveEvent);
     LOG_WITH_STREAM(MouseHandling, stream << "UIProcess: " << (didRemoveEvent ? "replaced" : "enqueued") << " mouse event " << event.type() << " (queue size " << internals().mouseEventQueue.size() << ")");
 
+    if (isMouseDown)
```

Comment:
Maybe any mouse event that's not a move?

Reply:
Makes sense — changed to that.